components/kyma-environment-broker/internal/process/deprovisioning/remove_runtime.go

```diff
@@ -43,16 +41,16 @@ func (s *RemoveRuntimeStep) Run(operation internal.DeprovisioningOperation, log
 		return s.operationManager.OperationFailed(operation, fmt.Sprintf("operation has reached the time limit: %s", RemoveRuntimeTimeout))
 	}
 
-	instance, err := s.instanceStorage.GetByID(operation.InstanceID)
```

Comment:
why did you change this code?

Reply:
We had the same code duplicated between the event_hub_step and this one that's why we moved the common code to a shared place

Reply:
but now it is not in the event_hub_step, the util function does not make sense.

Reply:
Yes, but maybe it will be needed in the future for new steps plus having the `Run()` more modular. What do you think?

Reply:
Reverted.